setup.py

```diff
@@ -216,7 +169,6 @@ def find_any_distribution(pkgs: List[str]) -> Optional[pkg_resources.Distributio
     },
     python_requires=">=3.6",
     install_requires=get_install_requires(),
-    tests_require=get_tests_require(),
```

Comment:
`tests_require` is deprecated.

https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies
https://github.com/pypa/setuptools/issues/1684